docs/sources/administration/configuration.md

```diff
@@ -450,203 +531,293 @@ Text used as placeholder text on login page for login/username input.
 
 Text used as placeholder text on login page for password input.
 
-<hr>
+### default_theme
 
-## [auth]
+Set the default UI theme: `dark` or `light`. Default is `dark`.
 
-Grafana provides many ways to authenticate users. The docs for authentication has been split in to many different pages
-below.
+### External user management
```

Comment:
Can skip this I think since it's mainly or only used for hosted grafana

Reply:
If it is visible to the user, then we should address it, even if it is just to say "only for hosted Grafana."

Reply:
Sure, but should not say "only for hosted Grafana.". This is how it's used on the Configuration -> Users page/in HG:
![image](https://user-images.githubusercontent.com/1668778/85403016-f9282d80-b55c-11ea-9bd0-cc7a9497956f.png)

Not really sure how to describe these settings.